src/articles/articles.controller.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Controller,
+  Get,
+  Put,
+  Delete,
+  Param,
+  ParseIntPipe,
+  Post,
+  UseGuards,
+  Body,
+  Query,
+} from '@nestjs/common';
+import { ArticlesService } from './articles.service';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { CreateArticleDto } from './dto/create-article.dto';
+import { GetUser } from 'src/auth/decorator/get-user.decorator';
+import { AuthGuard } from '@nestjs/passport';
+import { User } from 'generated/prisma';
+import { PaginationDto } from 'src/common/pagination.dto';
+
+@Controller('api/articles')
+export class ArticlesController {
+  constructor(private readonly articlesService: ArticlesService) {}
+
+  @Get('all')
```

Comment:
```suggestion
  @Get()
```
endpoint lấy ra list các article sẽ đặt tên `/api/articles` nhé,
như vậy là chuẩn RESTful rồi